### DISCUSSION TITLE
Some question about contribution.

### DISCUSSION THREAD

**Discussion by JiayuXu0** [open-mmlab/mmdetection#8902] [General]
I have fixed some warnings and added some features in open-mmlab project like openmmdetection and openmmcls. But  I have no idea what to do next. How can  I join a wechat group or qq group for communicating? The project committee can  assigned work task to me, like fix error or develop features.Thanks

Hello, 我已经尝试给openmmlab的项目提交了修改以及功能开发，但是我不知道下一步我可以为开源社区干点啥了。咱们有QQ群之类的么，那样项目管理者可以给我分配一些修复或者开发工作。谢谢

**Comment by RangeKing** [comment]
您好 @JiayuXu0。MMYOLO 是 OpenMMLab 下一个以社区为主导的开源库，非常欢迎有更多的开发者加入。有兴趣的话我们可以私聊联系。
